### ISSUE TITLE
Loss does not decrease for coco

### ISSUE THREAD

**Issue by strongwolf** [lzx1413/PytorchSSD#36]
Have you ever trained on coco for these models? I try to train coco with original SSD. However, the loss keeps increasing especially for the localization loss from 3 to 30! The accuracy also decreases. I use the default config and everything behaves well on Pascal voc.

**Comment by lzx1413** [comment]
I will try it later. Please keep pending.

**Comment by Chida15** [comment]
> Have you ever trained on coco for these models? I try to train coco with original SSD. However, the loss keeps increasing especially for the localization loss from 3 to 30! The accuracy also decreases. I use the default config and everything behaves well on Pascal voc.

Hello, I also meet this problem, have you solved it?

**Comment by strongwolf** [comment]
> > Have you ever trained on coco for these models? I try to train coco with original SSD. However, the loss keeps increasing especially for the localization loss from 3 to 30! The accuracy also decreases. I use the default config and everything behaves well on Pascal voc.
> 
> Hello, I also meet this problem, have you solved it?

The reason is that the augmentation operations have some errors that are not appropriate for some background images in coco.
